internal/pkg/manifest/rd_web_svc.go

```diff
@@ -28,6 +32,7 @@ type RequestDrivenWebServiceConfig struct {
 	InstanceConfig                    AppRunnerInstanceConfig `yaml:",inline"`
 	ImageConfig                       ImageWithPort           `yaml:"image"`
 	Variables                         map[string]string       `yaml:"variables"`
+	StartCommand                      StartCommand            `yaml:"start_command"`
```

Comment:
should we do `yaml:"command"` instead? to be consistent with our other workload types